src/cts/src/HTreeBuilder.cpp

```diff
@@ -363,103 +346,463 @@ void adjustToplevelTopology(Point<double>& a,
   }
 }
 
-bool HTreeBuilder::moveAlongBlockageBoundary(const Point<double>& parentPoint,
-                                             Point<double>& branchPoint,
-                                             double x1,
-                                             double y1,
-                                             double x2,
-                                             double y2)
+void HTreeBuilder::findLegalLocations(Point<double> parentPoint,
+                                      Point<double> branchPoint,
```

Comment:
Can these parameters be const&? Here and in the other functions with similar parameters.

Reply:
Yes, done for this and other modules.